binary_port/src/minimal_block_info.rs

```diff
@@ -6,13 +6,12 @@ use serde::{Deserialize, Serialize};
 
 #[cfg(test)]
 use rand::Rng;
-use schemars::JsonSchema;
```

Comment:
In my mind introducing schemars to binary_port is overkill just to save a few lines of code in sidecar so I removed it